_posts/blog/2024-08-30-accumulo4-preview.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Accumulo 4.0 Feature Preview"
+author: Dave Marion
+---
+
+## Background
+
+In version 2.1 we introduced two new optional and experimental features, [External Compactions](https://accumulo.apache.org/blog/2021/07/08/external-compactions.html) and [ScanServers](https://github.com/apache/accumulo/pull/2665). The ExternalCompactions feature included two new server processes, the CompactionCoordinator and the Compactor. Using these new processes and their related configurations allows the user to perform major compactions on Tablets external to the TabletServer process. The configuration in 2.1 allows the user to define different “queues” for the external compactions and to assign a “queue” to the Compactor process when it’s started. This provides the user with some capability to define the resources for different classes of compactions (see the referenced blog post for examples). ExternalCompactions may provide lower latency for major compactions because major compactions that run in the TabletServer may queue up with all of the major compaction threads are busy. 
```

Comment:
Just one comment, plan to look at this more another day:
Is the link to the ScanServer PR appropriate here? Seems like you have already written an in-depth comment on that PR that might be good for a blog post? Is that the plan and this is just temp?

Reply:
There is a [ticket](https://github.com/apache/accumulo/issues/4463) for a blog post for Scan Servers, it just hasn't been completed yet. When that is completed, then this reference could be changed.